### PR TITLE
Fixed README typo

### PR DESCRIPTION
**Pull request by scl2589**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,7 +166,7 @@ You can update properties dynamically at runtime. This can be used for a variety
...
 
-To understand how to change animation properties in Lottie, you should first understand how animation properties are stored in Lottie. Animation properties are stored in a data tree that mimics the information heirarchy of After Effects. In After Effects a Composition is a collection of Layers that each have their own timelines. Layer objects have string names, and their contents can be an image, shape layers, fills, strokes, or just about anything that is drawable. Each object in After Effects has a name. Lottie can find these objects and properties by their name using a KeyPath.
+To understand how to change animation properties in Lottie, you should first understand how animation properties are stored in Lottie. Animation properties are stored in a data tree that mimics the information hierarchy of After Effects. In After Effects a Composition is a collection of Layers that each have their own timelines. Layer objects have string names, and their contents can be an image, shape layers, fills, strokes, or just about anything that is drawable. Each object in After Effects has a name. Lottie can find these objects and properties by their name using a KeyPath.
```
Comment (hermet) [inline README.md:169]: ... looks identical. Can't figure out what did you change :)

Comment (hermet) [reply README.md:169]: got it!

Comment (scl2589) [reply README.md:169]: I have changed heirarchy to hierarchy!

Comment (scl2589) [reply README.md:169]: I have changed heirarchy to hierarchy!

Comment (scl2589) [reply README.md:169]: I have changed heirarchy to hierarchy!

Comment (scl2589) [reply README.md:169]: I have changed heirarchy to hierarchy!

Comment (scl2589) [reply README.md:169]: I have changed heirarchy to hierarchy!

**Pull request merged**